### PR TITLE
Network map

### PR DESCRIPTION
**Pull request by carsten19**
Implement a network map in a separate tab.
Uses the vis network library (loaded from remote).

### CODE REVIEW

**Comment by AlCalzone** [comment]
Care to post a screenshot here? I'm curious but I only have a production system with Z-Wave.

File: admin/i18n/de/translations.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "Add node": "Gerät hinzufügen",
     "Add node secure": "Sicheres Gerät hinzufügen",
     "Are you sure you want to remove all devices?": "Möchten Sie wirklich alle Geräte entfernen?",
+    "Associations": "Verbände",
```
Comment (AlCalzone) [inline admin/i18n/de/translations.json]: "Verknüpfungen" ist besser

File: admin/i18n/de/translations.json [hunk 2]
```diff
@@ -13,6 +14,7 @@
     "Last": "Letzte",
     "Logging:": "Logdatei schreiben:",
     "Manufacturer": "Hersteller",
+    "Network map": "Netzwerkkarte",
```
Comment (AlCalzone) [inline admin/i18n/de/translations.json]: Vielleicht besser "Netzwerk-Übersicht"? Netzwerkkarte klingt mir sehr nach der Hardware, die sich in meinem PC befindet.

File: admin/index.html [hunk 3]
```diff
@@ -3,6 +3,8 @@
 <link rel="stylesheet" type="text/css" href="../../lib/css/themes/jquery-ui/redmond/jquery-ui.min.css"/>
 <link rel="stylesheet" type="text/css" href="../../css/adapter.css"/>
 <link rel="stylesheet" type="text/css" href="../../lib/css/jquery.multiselect-1.13.css"/>
+<link href="https://unpkg.com/vis-network@latest/dist/vis-network.min.css" rel="stylesheet" type="text/css" />
```
Comment (AlCalzone) [inline admin/index.html]: That should be included locally (same for the js file). Just put it in the admin dir and reference it from there.

File: main.js [hunk 4]
```diff
@@ -192,6 +192,35 @@ function startAdapter(options) {
...
+                                    }
+                                }
+                                var channelID = calcName(nodeID);                                
+                                var item = {
+                                    "nodeID": nodeID,
+                                    "neighbors": edges[nodeID],
+                                    "label": objects[channelID].common.name
+                                }
+                                map.push(item);
+                                adapter.log.debug(nodeID+"="+JSON.stringify(item));
```
Comment (AlCalzone) [inline main.js]: remove debug log please

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by carsten19** [comment]
Bildschirmfoto 2019-08-25 um 21 04 40

**Comment by carsten19** [comment]
I just erased some labels. My map is pretty busy as it is production as well.
It uses the same network graphing as the zigbee adapter.

**Comment by Apollon77** [comment]
Cool

**Comment by AlCalzone** [comment]
That looks wild :D

**Pull request merged**